setup.sh

```diff
@@ -6,16 +6,39 @@ source ${BASH_SOURCE%/*}/vault_auth.sh
 echo "Installing Pip and required Python packages"
 apt-get install python3-pip -y > /dev/null
 pip3 install -q -r requirements.txt
+
+echo "-------------------------------------------"
+echo "Installing bb8 user and group"
+
 if ! id -u bb8 > /dev/null 2>&1; then
     useradd bb8
     password=$(vault read --field password secret/backup/bb8/user)
     echo "bb8:$password" | chpasswd
+
+    # set bb8 home dir
+    mkdir /var/lib/bb8/.ssh
+    usermod -m -d /var/lib/bb8 bb8
```

Comment:
Because of the directory above existing, this did not work for me (script dies with `usermod: directory /var/lib/bb8 exists`)

Consider:

```
    useradd bb8 -d /usr/lib/bb8
    password=$(vault read --field password secret/backup/bb8/user)
    echo "bb8:$password" | chpasswd

    # Perhaps defer until python?
    mkdir -p /var/lib/bb8/.ssh

    # add to docker group
    usermod -aG docker bb8
```